src/js/components/Select/SelectContainer.js

```diff
@@ -136,7 +136,7 @@ const SelectContainer = forwardRef(
         const clearButton = clearRef.current;
         if (onSearch) {
           const searchInput = searchRef.current;
-          if (searchInput && searchInput.focus) {
+          if (searchInput && searchInput.focus && !activeRef.current) {
```

Comment:
When I tested with only the change on line 118, it appeared to fix the issue so just wanted to know why this change was also necessary

Reply:
Ah gotcha, that make sense. Thanks for the explanation!